debs/SPECS/wazuh-manager/debian/rules

```diff
@@ -128,9 +128,12 @@ override_dh_install:
 	mkdir -p ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/darwin/19
 	mkdir -p ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/darwin/20
 	mkdir -p ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/darwin/21
+	mkdir -p ${TARGET_DIR}$(INSTALLATION_SCRIPTS_DIR)/sca/darwin/22
```

Comment:
File `etc/templates/config/darwin/22/sca.files` does not exist in the branch `master` of repository `wazuh/wazuh` yet